apps/meteor/lib/phoneValidator.ts

```diff
@@ -0,0 +1,6 @@
+export const validatePhone = (phone: string): boolean => {
+    const rfcPhoneRegex =/^[+]?[\s./0-9]*[(]?[0-9]{1,4}[)]?[-\s./0-9]*$/g;
```

Comment:
where did you get this regex?

Reply:
@debdutdeb from quera. It supports almost all international number.